akka-docs/src/main/paradox/stream/stream-integrations.md

```diff
@@ -81,14 +108,36 @@ the stream). The elements will be buffered until the stream can process them. Yo
 the queue and they will be emitted to the stream if there is demand from downstream, otherwise they will 
 be buffered until request for demand is received. 
 
+There are two means for sending messages to an Akka Stream. The first is to use Source.actorRef.
```

Comment:
I'm not sure if it's helpful to mention `Source.actorRef` in the section about `Source.queue` - this way we only show the 'preferred' way?

Reply:
Yeah it's a bit chaotic, I'm rephrasing those